jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -720,7 +720,18 @@ public void setBaseResource(Resource resourceBase)
 
     public void setBaseResource(Path path)
     {
-        setBaseResource(path == null ? null : ResourceFactory.root().newResource(path));
+        if (path == null)
+        {
+            // allow user to unset variable
+            setBaseResource((Resource)null);
+            return;
+        }
+
+        Resource resource = ResourceFactory.root().newResource(path);
```

Comment:
The `Path` may require mounting, right? So this may lead to a mount leak if I'm not mistaken.